crates/duty-tracker/src/tx_driver.rs

```diff
@@ -314,6 +314,10 @@ mod e2e_tests {
 
     use super::TxDriver;
 
+    // TODO(proofofkeags): once rust-bitcoin@0.33.x lands this isn't necessary anymore. This is
+    // due to a bug in rust-bitcoin.
+    pub(crate) const BIP34_MIN_BLOCKS: usize = 17;
```

Comment:
TxDriver should be moved into btc-notify. It's really an extension of the subscription system, not a component of the "duty tracker"